Jenkinsfile

```diff
@@ -27,12 +27,19 @@ def confluent_release_quality = choice(name: 'CONFLUENT_RELEASE_QUALITY',
     description: 'Determines the release extention (suffix) (ie: "prod" for public releases, "snapshot" for nightly builds)',
 )
 
+// Parameter for the molecule test scenario to run
+def molecule_scenario_name = choice(name: 'SCENARIO_NAME',
+    choices: ['plaintext-rhel', 'customcerts-rhel'],
+    defaultValue: 'plaintext-rhel',
+    description: 'The Ansible Molecule scenario name to run',
```

Comment:
customcerts rhel does not have the naming collisions logic in its molecule.yml file, will need this before you offer that as an option to jenkins

see plaintext-rhel/molecule.yml for reference

Reply:
I'll just make it a single choice here then.